var/spack/repos/builtin/packages/dihydrogen/package.py

```diff
@@ -155,7 +157,7 @@ def cmake_args(self):
             "-DH2_ENABLE_DISTCONV_LEGACY=%s" % ("+distconv" in spec),
             "-DH2_ENABLE_OPENMP=%s" % ("+openmp" in spec),
             "-DH2_ENABLE_FP16=%s" % ("+half" in spec),
-            "-DH2_ENABLE_HIP_ROCM=%s" % ("+rocm" in spec),
```

Comment:
Was this plain wrong, or the option changed? If the option changed this new recipe will break the build of older versions.

Reply:
@alalazo Good catch.  Let me fix that.

Reply:
Fixed.